homeassistant/components/sensor/netatmo.py

```diff
@@ -285,6 +286,8 @@ def update(self):
                 self._state = "High"
             elif data['wifi_status'] <= 55:
                 self._state = "Full"
+        elif self.type == 'lastupdated':
```

Comment:
Ugh, I missed this. We should not allow this. We never are allowed to store relative times in the state machine. Opening a PR to remove this feature.

Reply:
#15560